src/components/backgroundselector/BackgroundService.js

```diff
@@ -16,7 +16,6 @@ goog.require('ga_permalink');
   module.provider('gaBackground', function() {
     this.$get = function($rootScope, $q, gaTopic, gaLayers, gaPermalink,
         gaUrlUtils) {
-      var isOfflineToOnline = false;
```

Comment:
I think it's not used anymore.  It was used to not switch the bg saved offline to the default topic bg when we go back online, but I've tested and the bg is not switched when we go back online.

I think it became useless since we don't reload the topic anymore when we go back online.